docker/README.md

```diff
@@ -41,20 +41,25 @@ Here is an example to serve Resnet50 model using OpenVINO Integration with Tenso
 	
 	To run on default CPU backend:
 
-		$ docker run -t --rm -p 8501:8501 -v <path to Resnet50 model>:/models/resnet -e MODEL_NAME=resnet openvino_tensorflow/tensorflow-serving &
+		$ docker run -t --rm -p 8501:8501 -v <path to Resnet50 model>:/models/resnet -e MODEL_NAME=resnet openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
 
 	To run on iGPU:
 
-		$ docker run -t --rm -p 8501:8501 --device /dev/dri:/dev/dri -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=GPU openvino_tensorflow/tensorflow-serving &
+		$ docker run -t --rm -p 8501:8501 --device /dev/dri:/dev/dri -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=GPU openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
 
 	To run on Myriad:
 
-		$ docker run -t --rm -p 8501:8501 --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=MYRIAD openvino_tensorflow/tensorflow-serving &
+		$ docker run -t --rm -p 8501:8501 --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=MYRIAD openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
 
```

Comment:
How about VAD-M? Has TF Serving been tested on VAD-M?

Reply:
With the newly added docker run args, VAD-M backend is available